src/ember-app/app/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Although the Ember default is to provide an index file, as you've noted on that issue, Ember only really needs the configure that is included in the meta tag by default. If you have the configure built into the JS bundle, then Ember should boot and render into whatever div you tell it to, allowing you to integrate it with other systems.